confluent_kafka/avro/serializer/message_serializer.py

```diff
@@ -136,19 +140,14 @@ def encode_record_with_schema_id(self, schema_id, record, is_key=False):
         with ContextStringIO() as outf:
             # write the header
             # magic byte
-
             outf.write(struct.pack('b', MAGIC_BYTE))
 
             # write the schema ID in network byte order (big end)
-
             outf.write(struct.pack('>I', schema_id))
```

Comment:
this one and the write above should be condensed into one single struct.pack, for efficiency